src/components/Reports/MinisterHousingAllowance/Steps/StepThree/CalcComponents/Helper/CustomTextField.tsx

```diff
@@ -20,7 +20,8 @@ export const CustomTextField: React.FC<CustomTextFieldProps> = ({
   const currency = 'USD';
 
   const [focused, setFocused] = useState<string | null>(null);
-  const isEditing = (name: CustomTextFieldProps['name']) => focused === name;
+  const isEditing = (name: keyof CalculationFormValues & string) =>
+    focused === name;
```

Comment:
Let's make this into a function with brackets so it follows the repo's standards.